### PR TITLE
[feat] adding returned `stuff` from pages to `$page` store

### PR DESCRIPTION
**Pull request by tanhauhau**
### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [x] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpx changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: documentation/docs/03-loading.md [hunk 1]
```diff
@@ -153,4 +153,4 @@ If the `load` function returns a `props` object, the props will be passed to the
 
 This will be merged with any existing `stuff` and passed to the `load` functions of subsequent layout and page components.
 
-This only applies to layout components, _not_ page components.
+The final stuff will be available as part of the [page store](#modules-$app-stores)
```
Comment (benmccann) [inline documentation/docs/03-loading.md:156]:
```suggestion
The final stuff will be available as part of the [page store](#modules-$app-stores) providing a way to pass data upward to the layouts.
```

Comment (benmccann) [inline documentation/docs/03-loading.md:156]:
```suggestion
The final stuff will be available as part of the [page store](#modules-$app-stores) providing a way to pass data upward to the layouts.
```

**Review by benmccann — APPROVED** [review]

**Comment by PuruVJ** [comment]
Kudos naming it `stuff` 🤣👍

**Comment by gterras** [comment]
Thanks for this game changer feature :+1:

**Comment by ggjersund** [comment]
@tanhauhau I just started learning Svelte using Svelte-kit and I came across this issue with passing stuff from a child route to a parent layout file. I'm trying to pass a Load Error to a parent error file from the following path:
**src/routes/admin -> src/routes/__error.svelte**.

Should this work in you're case? or Do I have some sort of wrong implementation here?

```
<script context="module" lang="ts">
  export const load: Load<Input, Output> = async ({ params , fetch }) => {
    const url = `/admin/${params.index}.json`;
    const res = await fetch(url);
    if (res.ok) {
      return {
        props: {
          article: await res.json()
        }
      };
    }

    return {
      status: res.status,
      error: new Error(`Failed to find ${url}`),
      stuff: {
        page: "Admin"
      }
    };
  }
</script>
```

and then stuff should be retrievable from a parent `__error.svelte` page like this?
```
<script lang="ts">
    import { page } from '$app/stores';
</script>

<svelte:head>
  <title>ERROR PAGE</title>
</svelte:head>

<main>
  <h1>Failed to find {JSON.stringify($page.stuff)}</h1>
  <p>{`${$page.status}: ${$page.error.message}`}</p>
</main>
```

**Pull request merged**